docs/fundamentals/code-analysis/style-rules/ide0220.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "IDE0220: Add explicit cast"
+description: "Learn about code analysis rule IDE0220: Add explicit cast"
+ms.date: 07/20/2023
+ms.topic: reference
+f1_keywords:
+- IDE0220
+helpviewer_keywords:
+- IDE0220
+dev_langs:
+- CSharp
+---
+# Add explicit cast (IDE0220)
```

Comment:
I think it might add more clarity to explicitly mention that the rule is for foreach loops in the title, say `Add explicit cast in foreach loop`?